src/main/java/tasks/Deadline.java

```diff
@@ -0,0 +1,45 @@
+package tasks;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Deadline extends Task {
+
+    private String TaskIcon = "[D]";
```

Comment:
```suggestion
    private String taskIcon = "[D]";
```

 Variable names must be in camelCase.